src/internal/operators/mergeMap.ts

```diff
@@ -106,6 +105,12 @@ export function mergeMap<T, R, O extends ObservableInput<any>>(
       // The buffered values from the source (used for concurrency)
       let buffer: T[] = [];
```

Comment:
question: AFAICT, there's nothing here that removes values from the buffer upon explicit unsubscription. If a concurrency is specified, it's entirely possible that there could be a large number of buffered values (or a small number of values that are themselves large). In some other operator implementations, some consideration is given to releasing resources ASAP, but that does not seem to happen here.

Reply:
Good thinking. I can add a clean up to set `buffer` to `null` to ensure that array is GC'ed.